### PR TITLE
Update link to R-genomics lesson

### PR DESCRIPTION
**Pull request by luizirber**
The `R-genomics` repository doesn't have a master branch anymore, pointing to `gh-pages` now.

### CODE REVIEW

**Comment by luizirber** [comment]
Or maybe point to https://github.com/tracykteal/R-genomics, since it is the most updated version?

**Comment by hlapp** [comment]
@tracykteal any reason you don't want to bring the primary lesson repo up to date? I think it would be a bad idea to link to someone's (even of the ED's) fork as the authoritative one for a Data Carpentry lesson.

File: pages/lessons.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ These workshop materials are focused on working with genomics data.
 - [Know your data](https://github.com/datacarpentry/knowyourdata-genomics/tree/gh-pages/lessons)
 - [Introduction to the command line](https://github.com/datacarpentry/shell-genomics/blob/gh-pages/README.md)
 - [Introduction to cloud computing](https://github.com/datacarpentry/cloud-genomics/tree/master/lessons)
-- [Data analysis and visualization in R](https://github.com/datacarpentry/R-genomics/tree/master/lessons)
+- [Data analysis and visualization in R](https://github.com/datacarpentry/R-genomics/tree/gh-pages/lessons)
```
Comment (hlapp) [inline pages/lessons.md]: Shouldn't this consistently link to the README?

**Comment by tracykteal** [comment]
I need to bring the genomics lessons in the main repo up to date with the various forks. In the meantime, we should probably at least point to working links in the datacarpentry repo.

**Comment by tracykteal** [comment]
Also, they don't all currently have READMEs for that to be the standard link.

**Comment by tracykteal** [comment]
Thanks for this update @luizirber

**Pull request merged**